src/sections/Meshery/Features-section/index.js

```diff
@@ -67,10 +71,10 @@ const FeaturesSection = () => {
                   <img src={slide1} alt="Slide 1" />
                 </div>
                 <div>
-                  <img src={slide2} alt="Slide 2" />
+                  <StaticImage src={slide_2} alt="Slide 2" />
```

Comment:
Either use `StaticImage` for all, or for neither of them.

Reply:
One is gif so i can't use static Images on it.

Reply:
But that's not fixed, right?
The values for the slide images are passed as props, so they can be an image too.

Though, does it not work if you use `StaticImage` for a gif? Technically, the component should just bypass the processing and render it as it is.

Reply:
Yes, but when i used StaticImage for GIF it did not show anything.

Reply:
Ack.